lib/rialto/etl/extractors/sera.rb

```diff
@@ -10,6 +11,7 @@ module Extractors
       # Documentation: https://asconfluence.stanford.edu/confluence/display/MaIS/SeRA+API+-+User+Documentation
       class Sera
         include Rialto::Etl::Logging
+        class_attribute :oauth_client
```

Comment:
Do we need a `class_attribute` for this? I'm not seeing any code that uses this other than what's in `#client`. Would an instance variable suffice?

Reply:
Probably not, I can update.